synapse/rest/client/v1/room.py

```diff
@@ -349,6 +349,30 @@ async def inherit_depth_from_prev_ids(self, prev_event_ids) -> int:
 
         return depth
 
+    def _create_insertion_event_dict(self, sender: str, origin_server_ts: int):
+        """
+        Creates an event dict for an "insertion" event with the proper fields
+        and a random chunk ID.
+        Args:
+            sender: The event author MXID
+            origin_server_ts: Timestamp when the event was sent
+        Returns:
+            Tuple of event ID and stream ordering position
+        """
+
+        next_chunk_id = random_string(64)
```

Comment:
```suggestion
        next_chunk_id = random_string(8)
```

These will be scoped per room, so eight characters feels like more than enough? Which I think is equivalent to a ~50bit integer. 64 characters just seems needlessly long.